api/IDBDatabase.json

```diff
@@ -165,6 +165,193 @@
           }
         }
       },
+      "abort_event": {
+        "__compat": {
+          "description": "<code>abort</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBDatabase/abort_event",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
+            ],
+            "chrome_android": {
+              "version_added": true
```

Comment:
> Leave webkit version number out since it would never have shipped on Android.
> Please change throughout.

Just to clarify: you mean remove the `webkit` prefix clause for:
* `chrome_android`
* `samsunginternet_android`
* `webview_android`

...but keep it for `chrome`?